website/docs/r/search_service.html.markdown

```diff
@@ -54,3 +54,11 @@ The following arguments are supported:
 The following attributes are exported:
 
 * `id` - The Search Service ID.
+
+## Import
+
+Search Services can be imported using the `resource id`, e.g.
```

Comment:
We can do, but we should update all of them to match this, as this was C&P'd from an existing page for consistency ([example](https://www.terraform.io/docs/providers/azurerm/r/eventhub_namespace.html#import))